### PR TITLE
Fixes readme on npm

### PR DESCRIPTION
**Pull request by dbrudner**
Here's a screenshot of the log for this:

image

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ jest.*.js
 Makefile
 webpack.config.js
 docs
+README.md
```
Comment (chrissrogers) [inline .npmignore:12]: I think npmignore needs to just ignore README.mdx, no?

File: package.json [hunk 2]
```diff
@@ -49,7 +49,9 @@
   },
   "scripts": {
     "test": "jest",
-    "prepublishOnly": "babel lib -d lib-dist",
+    "prepublishOnly": "babel lib -d lib-dist && cp README.mdx README.md",
+    "postpublish": "rm README.md",
+    "post": "babel lib -d lib-dist && cp README.mdx README.md",
```
Comment (chrissrogers) [inline package.json:54]: Looks like this was left dangling?

Comment (dbrudner) [reply package.json:54]: ah yeah, I rushed this one.

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,4 @@ lib-dist
 node_modules
 
 npm-debug.log
+README.md
```
Comment (dbrudner) [inline .gitignore:8]: I'm not sure we want to add this. It'd be an annoyance if/when we update the readme.

**Pull request merged**